src/pages/projects.tsx

```diff
@@ -80,15 +81,26 @@ function toProjectTypeFilter(category?: string | null): ProjectTypeFilter | unde
 
   return index !== -1 ? categories[index] : undefined
 }
+function toDateFilter(value: string | null): number | undefined {
+  if (value) {
+    const parsedValue = Number(value)
+    if (!isNaN(parsedValue)) {
+      return parsedValue
+    }
+  }
+  return undefined
+}
 
 export default function ProjectsPage() {
   const t = useFormatMessage()
   const params = useURLSearchParams()
   const type = toProjectTypeFilter(params.get('type'))
   const status = toProjectStatus(params.get('status'))
   const subtype = toGrantSubtype(params.get('subtype'), () => undefined)
+  const year = toDateFilter(params.get('year'))
+  const quarter = toDateFilter(params.get('quarter'))
 
-  const { projects, isLoadingProjects } = useProjects()
+  const { projects, isLoadingProjects } = useProjects(year && quarter ? { year, quarter } : undefined)
```

Comment:
If year and quarter could be undefined and checks are added in the hook I'd just write here `useProjects({ year, quarter })`